test/DynamoCoreWpfTests/NodeViewCustomizationTests.cs

```diff
@@ -200,8 +200,8 @@ public void PythonNodeHasButtonsAndCorrectNumberOfInputs()
             var eles = nodeView.inputGrid.ChildrenOfType<DynamoNodeButton>();
```

Comment:
Test case updated due to name changes from removing `DockPanel` above. No real code change.